src/mod/applications/mod_conference/conference_loop.c

```diff
@@ -1593,10 +1593,10 @@ void conference_loop_output(conference_member_t *member)
 		if (conference_utils_member_test_flag(member, MFLAG_INDICATE_UNMUTE)) {
 			if (!zstr(member->conference->unmuted_sound)) {
 				conference_member_play_file(member, member->conference->unmuted_sound, 0, SWITCH_TRUE);
-			} else {
+			} else if (!zstr(member->conference->unmuted_tts)) {
 				char msg[512];
 
-				switch_snprintf(msg, sizeof(msg), "Un-Muted");
+				switch_snprintf(msg, sizeof(msg), member->conference->unmuted_tts);
 				conference_member_say(member, msg, 0);
```

Comment:
Here too
```
else {
	char msg[512];
	switch_snprintf(msg, sizeof(msg), "Un-Muted");
	conference_member_say(member, msg, 0);
}
```

Reply:
No, they are needed to be configured in the profile instead if we want the profile to play TTS unmuted sound